### ISSUE TITLE
Imports Issue

### ISSUE THREAD

**Issue by berenmacbowman** [JonathanLink/PDFLayoutTextStripper#13]
org.apache.pdfbox.io IMPORTS aren't working for me;
where do I download them and get them to work?

**Comment by JonathanLink** [comment]
Hello, as mentioned in the readme pdfbox is available there:
https://mvnrepository.com/artifact/org.apache.pdfbox/pdfbox/2.0.6
If you don't use Maven to install pdfbox library, you will also need to download this two sub-dependencies: 
https://commons.apache.org/proper/commons-logging/download_logging.cgi
https://mvnrepository.com/artifact/org.apache.pdfbox/fontbox

**Comment by berenmacbowman** [comment]
The imports show no errors now, but when I click run nothing happens. 
 
I followed those links. First I downloaded the Binaries by clicking commons-logging-1.2-bin.tar.gz &  commons-logging-1.2-bin.zip; then I did the same for Source. WHAT does the md5 and pgp mean? I just ignored them. 
 
THEN I went to your next link and downloaded the 2.0.6 bundle. 
 
I saved all those downloaded files into one folder. They look weird; there are two GZ Files and two folders that say compressed. FONTBOX-2.0.6 and PDFBOX-2.0.6 are also in there, and those are EXECUTABLE JAR files. Is that all correct? 
 
THEN in Eclipse I right-clicked PDFLayoutTextStripper.java, clicked CONFIGURE BUILD PATH, went to LIBRARIES, clicked ADD EXTERNAL JARS. FONTBOX and PDFBOX showed up, and so did the .zip folders. NO GZ FILES anymore. 
 
I opened all of them; clicked Apply and Okay. 
 
Then I ran it...nothing happened."
